File: app/src/main/java/com/babestudios/companieshouse/data/model/company/RegisteredOfficeAddress.java

package com.babestudios.companieshouse.data.model.company;

import com.google.gson.annotations.SerializedName;

import javax.annotation.Generated;

public class RegisteredOfficeAddress {

	@SerializedName("address_line_1")
	public String addressLine1;

	@SerializedName("address_line_2")
	public String addressLine2;

	@SerializedName("care_of")
	public String careOf;

	@SerializedName("country")
	public String country;

	@SerializedName("locality")
	public String locality;

	@SerializedName("po_box")
	public String poBox;

	@SerializedName("postal_code")
	public String postalCode;

	@SerializedName("premises")
	public String premises;

	@SerializedName("region")
	public String region;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/RegisteredOfficeAddress.kt
package com.babestudios.companyinfouk.data.model.company

import com.google.gson.annotations.SerializedName

import javax.annotation.Generated

class RegisteredOfficeAddress {

	@SerializedName("address_line_1")
	var addressLine1: String? = null

	@SerializedName("address_line_2")
	var addressLine2: String? = null

	@SerializedName("care_of")
	var careOf: String? = null

	@SerializedName("country")
	var country: String? = null

	@SerializedName("locality")
	var locality: String? = null

	@SerializedName("po_box")
	var poBox: String? = null

	@SerializedName("postal_code")
	var postalCode: String? = null

	@SerializedName("premises")
	var premises: String? = null

	@SerializedName("region")
	var region: String? = null

}
